### PR TITLE
Fixing some links and a typo.

### PR DESCRIPTION
**Pull request by arturgontijo**
Small fixes that I noticed while re-doing all the tutorials.

### CODE REVIEW

File: v3/tutorials/08-ink-workshop/a-getting-started/index.mdx [hunk 1]
```diff
@@ -130,10 +130,10 @@ The `cargo-contract` package provides a command-line interface for working with
 
 1. Open a terminal shell on your computer.
 
-1. Install `dylint-link`, required to lint ink! contracts, warning you about things like using API's in a way that could lead to security issues.
+1. Install `cargo-dylint` and `dylint-link`, required to lint ink! contracts, warning you about things like using API's in a way that could lead to security issues.
```
Comment (NukeManDan) [inline v3/tutorials/08-ink-workshop/a-getting-started/index.mdx:133]: This was not explicitly needed for me, it was for you?

File: v3/tutorials/07-add-a-pallet/index.mdx [hunk 2]
```diff
@@ -395,7 +395,7 @@ To return the information stored for Alice:
 
 1. Select `nicks` from the list of pallets available to query.
 
-1. Select the [`nameOf`](/rustdocs/latest/pallet_nicks/pallet/enum.Call.html#variant.set_name).
+1. Select the [`nameOf`](/rustdocs/latest/src/pallet_nicks/lib.rs.html#113).
```
Comment (NukeManDan) [inline v3/tutorials/07-add-a-pallet/index.mdx:398]: It's preferable to point to the rustdocs not the source hosted by rustdocs, or the source instead on github.

**Review by NukeManDan — CHANGES_REQUESTED** [review]
Thanks! just a couple nits

File: v3/tutorials/08-ink-workshop/a-getting-started/index.mdx (hunk 1 above)
Comment (arturgontijo) [reply v3/tutorials/08-ink-workshop/a-getting-started/index.mdx:133]: Just ran all the installation steps in a Docker container with a fresh `ubuntu:18.04` image and got this:
```
root@gontijo:/opt/flipper# cargo +nightly contract build
 [1/5] Checking ink! linting rules
ERROR: cargo-dylint was not found!
Make sure it is installed and the binary is in your PATH environment.

You can install it by executing `cargo install cargo-dylint`.root@gontijo:/opt/flipper# cargo +nightly contract build
 [1/5] Checking ink! linting rules
ERROR: cargo-dylint was not found!
Make sure it is installed and the binary is in your PATH environment.

You can install it by executing `cargo install cargo-dylint`.
```

I also see that dep in [cargo-contract#installation](https://github.com/paritytech/cargo-contract#installation).

File: v3/tutorials/07-add-a-pallet/index.mdx (hunk 2 above)
Comment (arturgontijo) [reply v3/tutorials/07-add-a-pallet/index.mdx:398]: Agree. But I wasn't able to find a reference for the `nameOf` Storage there =/.
Right now the link points to [enum.Call.html#variant.set_name](https://docs.substrate.io/rustdocs/latest/pallet_nicks/pallet/enum.Call.html#variant.set_name) so maybe we should keep it as it is.

File: v3/tutorials/08-ink-workshop/a-getting-started/index.mdx (hunk 1 above)
Comment (NukeManDan) [reply v3/tutorials/08-ink-workshop/a-getting-started/index.mdx]: yes I know it's specified upstream, but I don't tihnk it's explicitly needed cc @cmichi can you chime in here? 
Related to #1044

**Comment by NukeManDan** [comment]
`1. Replace the default template source code with new [erc20](https://docs.substrate.io/assets/tutorials/ink-workshop/erc-template-lib-0.rs) source code.`
and a few other code files are not present. Needs to be added to source, and reference github directly as a stop gap, not rendered/hosted on the docs website.

**Comment by NukeManDan** [comment]
I _think_ this is all fixed on the new `main` - please lmk if not!

**Pull request closed without merging**